src/index.html

```diff
@@ -9,6 +9,49 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#home" class="logo">
+        <img src="./images/logo.png" alt="Logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="list__item">
+            <a href="#apple" class="nav__link is-active">APPLE</a>
+          </li>
+
+          <li class="list__item">
+            <a href="#samsung" class="nav__link">SAMSUNG</a>
```

Comment:
Don't use uppercase letters, let's find out how to do it in the CSS
```suggestion
            <a href="#samsung" class="nav__link">samsung</a>
```